server/app/services/cloud/websocket_client.rb

```diff
@@ -26,13 +26,22 @@ def self.api_uri
 
     include Logging
     KEEPALIVE_TIME = 30
+
+    if defined? Faye::Websocket::Client.CLOSE_TIMEOUT
```

Comment:
We should update to faye-websocket 0.10.7, so that we don't need this monkey-patching.